lib/install/.circleci/config.yml

```diff
@@ -121,17 +121,22 @@ jobs:
 
   upgrade_tools:
     docker:
-      - image: circleci/ruby:2.6.4-stretch-node
+      - image: circleci/ruby:2.6-stretch-node
         environment:
           DISABLE_SPRING: 1
           BUNDLE_GEMFILE: Gemfile.tools
           BUNDLE_JOBS: 3
           BUNDLE_RETRY: 3
+
+          # Provide values for current repository
+          REPOSITORY_OWNER: jetthoughts
+          REPOSITORY_TITLE: jt_tools
```

Comment:
we run installer for some org and some repo. we does not know org and repo now. This is passed by CI

Reply:
Changed